packages/calypso-analytics/src/utils/get-tracking-prefs.ts

```diff
@@ -0,0 +1,102 @@
+import cookie from 'cookie';
+import isCountryInGdprZone from './is-country-in-gdpr-zone';
+import isRegionInCcpaZone from './is-region-in-ccpa-zone';
+
+export const TRACKING_PREFS_COOKIE_V1 = 'sensitive_pixel_option';
+export const TRACKING_PREFS_COOKIE_V2 = 'sensitive_pixel_options';
+
+export type TrackingPrefs = {
```

Comment:
We have these types here `TrackingPrefs` client/lib/analytics/utils/get-tracking-prefs.ts, is this to move them to a package? If we do, could we also clean up the ones in client?

Reply:
Hey @leomattic! I would be happy to help with that one. You indeed cannot import TrackingPrefs from `client/...`, but we could do it the other way around - it's just so as to not repeat the code.

You can replace uses of TrackingPrefs with `import { TrackingPrefs } from '@automattic/calypso-analytics'`.